game-core/src/main/java/games/strategy/engine/lobby/moderator/toolbox/tabs/Pager.java

```diff
@@ -0,0 +1,52 @@
+package games.strategy.engine.lobby.moderator.toolbox.tabs;
+
+import java.util.List;
+import java.util.function.Function;
+
+import org.triplea.http.client.moderator.toolbox.PagingParams;
+
+import lombok.Builder;
+
+/**
+ * Class to take care of paging details. Keeps track of current row, when
+ * requesting data will increment the current row to request the next row
+ * on the next data fetch.
+ */
+public class Pager {
```

Comment:
This class was to avoid re-writing the same test code, @RoiEXLab, I think this is what you were looking for in terms of client-side support  : )